frontend/src/components/results/FeedbackPopup.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Subscribe from '../special/Subscribe';
+import { SecondaryHeaderText } from '../core/Text';
+import { SecondaryHeaderTextLink } from '../core/Link';
+
+function FeedbackPopup() {
+  return (
+    <div>
+      <h1>
+        <span role="img" aria-label="wave">Hello! 👋</span>
+      </h1>
+      <SecondaryHeaderText>
+        Makers of CodeJoust here - if you like (or dislike) the platform,
+        we would love to hear your feedback! Feel free to
+        {' '}
+        <SecondaryHeaderTextLink to="/contact-us" target="_blank">contact us</SecondaryHeaderTextLink>
```

Comment:
Perhaps make this a bit more evident that it's a link? Maybe changing the color to the blue link color used on the contact-us page, or something a bit similar? At first glance this is just text.